engine/access/rpc/backend/transactions/transactions_test.go

```diff
@@ -440,7 +440,7 @@ func (suite *Suite) TestGetSystemTransaction_HappyPath() {
 	require.NoError(suite.T(), err)
 
 	block := unittest.BlockFixture()
-	res, err := txBackend.GetSystemTransaction(context.Background(), block.ID())
+	res, err := txBackend.GetSystemTransaction(context.Background(), flow.ZeroID, block.ID())
```

Comment:
can you also add tests for the case where an ID is passed and the tx fails with `codes.Unavailable`